Automattic/wp-calypso
# Unit Tests: Remove Theme Card's More Menu Tests

Based on #87004
Related to #87014

## Proposed Changes

Removes the unit tests related to the Theme card's More menu.
@Automattic/t-rex is removing it as part of the Theme Tiers v1 project (see https://github.com/Automattic/dotcom-forge/issues/4597).

## Testing Instructions



* Run `yarn test-client client/components/theme/test/index.jsx`.
* Ensure all tests pass.

## Pre-merge Checklist
